mapswipe_workers/tests/unittests/test_create_project.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import set_up
+import tear_down
+from click.testing import CliRunner
+from mapswipe_workers import auth, mapswipe_workers
+
+
+class TestCreateProject(unittest.TestCase):
+    def setUp(self):
+        self.project_id = set_up.create_test_project_draft("build_area")
+
+    def tearDown(self):
+        tear_down.delete_test_data(self.project_id)
+
+    def test_create_project(self):
+        runner = CliRunner()
+        runner.invoke(mapswipe_workers.run_create_projects)
+
+        pg_db = auth.postgresDB()
+        query = "SELECT * FROM projects WHERE project_id = '{0}'".format(
+            self.project_id
+        )
+        result = pg_db.retr_query(query)
+        self.assertIsNone(result)
+
+        fb_db = auth.firebaseDB()
+        ref = fb_db.reference("/v2/projects/{0}".format(self.project_id))
+        result = ref.get()
+        self.assertIsNone(result)
+
```

Comment:
just for my understanding: here we could add more tests, e.g. test_create_project_invalid_geometry, correct?

Reply:
Yes. And we probably should add more tests.

Right now only minimal tests are run.

One goal is it to at least run mapswipe commands (e.g. archive) with test data using scripts for easier development.